### PR TITLE
Add cd commands

### PR DESCRIPTION
**Pull request by griffinhadfield**
Addressing https://github.com/openliberty/guides-common/issues/514

### CODE REVIEW

**Review by tt-le — APPROVED** [review]
LGTM checked on LGDEV

**Review by tt-le — CHANGES_REQUESTED** [review]
Upon further thought,  I think we should reword the text preceding the `cd start` command. Right now it seems like we're asking the user to navigate to the `start` directory and running `cd start` again.

**Comment by griffinhadfield** [comment]
@tt-le I have changed that line to the following:

> Navigate to the `start` directory build the applications by running the following commands:

What do you think?

File: README.adoc [hunk 1]
```diff
@@ -148,9 +148,10 @@ https://openliberty.io/guides/containerize.html[Containerizing Microservices^] g
 which covers Dockerfiles in depth.
 
 
-Navigate to the `start` directory and run the following command to build the applications:
+Navigate to the `start` directory build the applications by running the following commands:
```
Comment (tt-le) [inline README.adoc:151]: @griffinhadfield minor mistake
> Navigate to the `start` directory **and** build the applications by running the following commands:

**Review by tt-le — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (griffinhadfield) [reply README.adoc]: Good catch, thank you. Its now fixed

**Review by tt-le — APPROVED** [review]
LGTM

**Pull request merged**